Orange/widgets/data/owtable.py

```diff
@@ -69,24 +74,157 @@ def __init__(self, sourcedata, parent=None):
         self._labels = list(sorted(
             {label for label in labels if not label.startswith("_")}))
 
+        self.Y_dim = len(self.domain.class_vars)
+        self.M_dim = len(self.domain.metas)
+
+        self.cache = LRUCache(10000)
+        self.viewport = Viewport(0, 0, 0, 0)
+
+        self.view = view  # ugly but I need it to know what to cache
+
+        self.timer = QTimer()
```

Comment:
this timer should have a better name :)

Reply:
I can't come up with anything more descriptive. Any ideas?

Reply:
Perhaps `delayed_update`?